File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ride/StartRideResponse.java
package com.lattis.ellipse.data.network.model.response.ride;

import com.google.gson.annotations.SerializedName;

public class StartRideResponse {

    @SerializedName("payload")
    StartRideDataResponse startRideDataResponse;

    public StartRideDataResponse getStartRideDataResponse() {
        return startRideDataResponse;
    }

    @Override
    public String toString() {
        return "GetUserResponse{" +
                "StartRideDataResponse=" + startRideDataResponse +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/ride/StartRideResponse.kt
package com.lattis.data.entity.response.ride

import com.google.gson.annotations.SerializedName

class StartRideResponse {
    @SerializedName("payload")
    var startRideDataResponse: StartRideDataResponse? = null

    override fun toString(): String {
        return "GetUserResponse{" +
                "StartRideDataResponse=" + startRideDataResponse +
                '}'
    }
}